tests/sudorule/test_sudorule.yml

```diff
@@ -123,6 +305,125 @@
     register: result
     failed_when: result.changed
 
+  # rafasgj
```

Comment:
What is `rafasgj`?

Reply:
Myself. It's a note left behind. Sorry for that.